.github/workflows/ci.yml

```diff
@@ -109,6 +109,11 @@ jobs:
           TARGET: L1MetadataArray_test
           STAGES: synth_sdc synth floorplan place generate_abstract
         run: .github/scripts/build_local_target.sh
+      - name: open targets
+        run: |
+          for stage in "synth" "floorplan" "place"; do
```

Comment:
is "set -e" set here?

Reply:
Yes, it is set by default in each script from `run`